### PR TITLE
gitsign: Bump to v0.7.1

### PR DESCRIPTION
**Pull request by wlynch**
#### Summary

gitsign: Bump to v0.7.1

#### Release Note

Updated gitsign to v0.7.1

### CODE REVIEW

**Comment by wlynch** [comment]
cc @cpanato @priyawadhwa

File: Formula/gitsign-credential-cache.rb [hunk 1]
```diff
@@ -4,26 +4,26 @@
...
-      sha256 "34b92bb4349af1ed1820f95c20241f9c3c424b92869ca8c16711c07f3baa34be"
+      sha256 "5eacdd6fa5e6b4631162a1cba0133d1d45d3fa0c21b14d1519eafaafc543db70"
     end
   end
 
   on_linux do
     if Hardware::CPU.intel?
       url "https://github.com/sigstore/gitsign/releases/download/v#{version}/gitsign-credential-cache_#{version}_linux_amd64"
-      sha256 "4dc410517c92db16f7bd7e382d0b2b0ba68b8500a90164612ead284d85140dbe"
+      sha256 "7bbf3924d8684ef6a39ecefa8de773b99212cde94b5ea379dfd7a58afa5704b5"
```
Comment (jkjell) [inline Formula/gitsign-credential-cache.rb:26]: I think you missed linux_arm64: ccd1a0a66dc47decf64cee54306209891060ba62eba0fd64107319fc547759fe

Comment (wlynch) [reply Formula/gitsign-credential-cache.rb:26]: Forgot to save the file 🤦  nice catch!

The unit tests usually catch stuff like this. I wonder why it didn't this time 🤔  /cc @cpanato

**Review by jkjell — APPROVED** [review]
Thanks Billy!

File: Formula/gitsign-credential-cache.rb (hunk 1 above)
Comment (cpanato) [reply Formula/gitsign-credential-cache.rb:26]: I will take a look

**Review by cpanato — APPROVED** [review]

**Pull request merged**